Paws4Me/Paws4Me/UserInterfaces/styleKit.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  styleKit.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/08.
+//
+
+import Foundation
+
+import UIKit
+
+extension UIColor {
+    static var myAppPurple: UIColor {
+        return  UIColor(red: 55/255, green: 21/255, blue: 67/255, alpha: 1)
```

Comment:
So unless your app is going to handle dark and light mode the same way this should actually be in the assets folder 😄 so that you can account for both variations.

Reply:
Just to clarify if I have an asset called primaryPurple in my assets folder can I say:

   button.layer.borderColor = UIColor.init(named: "primaryPurple")?.cgColor

Reply:
No so it would still live here in the `stykeKit`

```
extension UIColor {
   static var primaryColour: UIColor {
      UIColor(named: primaryPurple) ?? UIColor.clear
   }
}
```

and then you would still access this the same:

`button.layer.borderColor = UIColor.primaryColour.cgColor`

Reply:
Thank you for the explanation. I really appreciate it 👍🏻